src/Controller/Frontend/ListingController.php

```diff
@@ -80,6 +80,16 @@ public function listing(ContentRepository $contentRepository, string $contentTyp
 
         $records = $this->setRecords($content, $amountPerPage, $page);
 
+        // Set canonical URL
+        $this->canonical->setPath(
+            'listing_locale',
+            array_merge([
+                'contentTypeSlug' => $contentType->get('slug'),
+                '_locale' => $this->request->getLocale(),
+            ], $queryParams),
```

Comment:
Only in PHP8 you are allowed to have trailing comma's is parameter lists. :-)

```suggestion
            ], $queryParams)
```

Reply:
Thx! It's been a couple of years I don't code in PHP… that's I suppose is TS habit. ^_^U